### PR TITLE
Documentation updates.

### PR DESCRIPTION
**Pull request by grigi**
Hi

This pull request Fixes .event instead of .events for AMQP messages.
It also updated the release notes, and expands a bit on AMQP integration.

### CODE REVIEW

File: docs/amqp-integration.rst [hunk 1]
```diff
@@ -0,0 +1,48 @@
+.. _amqp-integration:
+
+AMQP integration
+================
+
+Should you choose to use the AMQP queues to handle messaging with Junebug, when
+you specify the ``amqp_queue`` parameter when you configure the
+:ref:`channel <channels>`, the usage is as follows:
+
+* Inbound *(mobile originated)* messages are sent to the **key** of
```
Comment (rudigiesler) [inline docs/amqp-integration.rst]: Perhaps we should call it the `routing key` instead of `key`?

**Comment by rudigiesler** [comment]
Just that one comment from me, otherwise it looks good. Thanks for updating this :)

**Comment by grigi** [comment]
@rudigiesler Thanks, I did your suggestion. It definitely avoids some ambiguity :-)

**Comment by rudigiesler** [comment]
:+1:

**Pull request merged**